octave/fsk_cml.m

```diff
@@ -12,7 +12,7 @@
 %randn('seed',1);
 format short g 
 more off
-init_cml('~/cml/');
+init_cml(getenv("CMAKE_CML_PATH"));
 

```

Comment:
This is a good solution for finding the unknown location of `$(CMAKE_CURRENT_BINARY_DIR}/cml`.  However we also need a way run this scripts from the Octave CLI during development.  Perhaps one solution is to set the environment variable in the developers `~/octave.rc`  After all, there are very few developers of this code (just myself and @srsampson AFAIK)

Reply:
Hmm, be nice not to set it system wide though, e.g. I often run several codec2 repos at various stages of development.  OK this works - just setup a `.octavetc` file in your `codec2/octave` directory:
```
setenv("CMAKE_CML_PATH","../build_linux/cml")
```